src/python/pants/rules/core/fmt_test.py

```diff
@@ -110,8 +110,9 @@ def test_single_target(self) -> None:
     self.assert_workspace_modified()
 
   def test_multiple_targets(self) -> None:
-    # NB: we do not test the case where FmtResults have conflicting changes, as that logic
-    # is handled by DirectoriesToMerge.
+    # NB: we do not test the case where AggregatedFmtResults have conflicting changes, as that
```

Comment:
I'm not sure I understand how `DirectoriesToMerge` handles the conflicting change problem, since `DirectoriesToMerge` just raises an exception on overlapping files?